migrations/1623403492949-ldap-sync-idx.js

```diff
@@ -0,0 +1,20 @@
+// eslint-disable-next-line no-unused-vars
+const { alert, error } = require('../src/logger');
```

Comment:
migration for sync can removed, because npm run syncIndex can do the same. Also the change in backup/setup/*